src/Kafka/CreateKafkaInstance/CreateKafkaInstance.tsx

```diff
@@ -289,18 +291,14 @@ export const CreateKafkaInstance: FunctionComponent<
           />
         </FlexItem>
       </Flex>
-      <Flex>
```

Comment:
Changed this layout to fix the scroll happening inside the modal even when there is enough space not to have a scrollbar